### ISSUE TITLE
Separating metadata creation from rest of the package

### ISSUE THREAD

**Issue by EveliinaP** [LivingNorway/LivingNorwayR#55]
Would it be an idea to separate the part of the package that helps in creating EML as its own package that could be used independently of rest of the package?

**Comment by DrMattG** [comment]
I agree that the EML part can be a useful part of non-livingNorwayR workflows but I am not sure that there is a need to seperate it out. One could achieve interaction with eml using LNR functions without needing to go through the whole workflow. Can you forsee a need for having a package rather than just calling in the LNR functions?

**Comment by EveliinaP** [comment]
I was mainly thinking that the functions might be a bit "hidden" to people who might not want to do the whole workflow so if you wanted more people to make use of them then it could be useful to have it separate. Also, I think maybe Joe was saying that the package was getting bit so it would have been one way of dividing it into smaller bits. But I have no problem with using the functions as they are now in the package.
